packages/frontend/src/scripts/forced-actions/state.ts

```diff
@@ -1,22 +1,19 @@
 import { AssetId } from '@explorer/types'
 
 import { formatCurrencyInput } from '../../utils/formatting/formatCurrencyInput'
-import { ForcedActionFormProps } from '../../view/pages/forced-actions/ForcedActionFormProps'
+import { NewForcedActionFormProps } from '../../view/pages/forced-actions/NewForcedActionFormProps'
 import { FormAction, FormState } from './types'
-import { getAsset, getFormType, parseCurrencyInput } from './utils'
+import { getFormType, parseCurrencyInput } from './utils'
 
-export function getInitialState(
-  props: ForcedActionFormProps,
-  queryString: string
-): FormState {
-  const search = new URLSearchParams(queryString)
-  const assetId = search.get('assetId')
-  const initialAssetId = assetId ? AssetId(assetId) : props.selectedAsset
-  const selectedAsset = getAsset(initialAssetId, props.assets)
-  const type = getFormType(selectedAsset)
+export function getInitialState(props: NewForcedActionFormProps): FormState {
```

Comment:
Are we using this for forced spot withdrawals? I don't think so but I want to make sure

Reply:
Nope, thats only for perpetuals